luaui/RmlWidgets/rml_context_manager.lua

```diff
@@ -0,0 +1,50 @@
+if not RmlUi then
+    return
+end
+
+function widget:GetInfo()
+    return {
+        name = "Rml context manager",
+        desc = "This widget is responsible for handling dynamic interactions with Rml contexts.",
+        author = "Mupersega",
+        date = "2025",
+        license = "GNU GPL, v2 or later",
+        layer = 0,
```

Comment:
This should go on layer `-1000000` if it needs to always be executed before any other rmlui widget